cms/djangoapps/contentstore/views/course.py

```diff
@@ -1024,6 +1024,8 @@ def grading_handler(request, course_key_string, grader_index=None):
 def _refresh_course_tabs(request, course_module):
     """
     Automatically adds/removes tabs if changes to the course require them.
+
+    Raises an InvalidTabsException if there's a problem with the new version of the tabs.
```

Comment:
Can you use the Google docstring convention here:

    Raises:
        InvalidTabsException: Raised if there's a problem with the new version of the tabs.

See: http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html